python/cuml/common/cuda.pyx

```diff
@@ -16,6 +16,14 @@
 
 # distutils: language = c++
 
+
+from rmm._cuda.gpu import getDeviceCount
+
+
+def has_cuda_gpu():
```

Comment:
For consistency with conventions from PyTorch/Tensorflow, would consider naming this `is_cuda_available`.

https://pytorch.org/docs/stable/generated/torch.cuda.is_available.html#torch.cuda.is_available

https://www.tensorflow.org/api_docs/python/tf/test/is_gpu_available